examples/kubernetesevents_scaledobject.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: keda.k8s.io/v1alpha1
+kind: ScaledObject
+metadata:
+  name: kubernetes-events-scaledobject
+  namespace: keda-kubernetes-events-scaler
+  labels:
+    deploymentName: keda-kubernetes-events-scaler-node
+spec:
+  scaleTargetRef:
+    deploymentName: keda-kubernetes-events-scaler-node
+  triggers:
+  - type: kubernetes-events
+    metadata:
+      fieldSelector: "metadata.namespace=events-sample,type=Warning" # Optional
```

Comment:
does empty mean all?

Reply:
Yes, by default everything is selected

```sh
kubectl get events --field-selector="" --all-namespaces
```

https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/

Reply:
How will it react if I don't have permissions for all?

Reply:
Do you mean if the Keda service account doesn't have permissions? It will return an empty list. I'm using the standard client-go api.

Reply:
> At the moment we're giving cluster-admin permissions to the service account.

Sure, but we cannot assume that

> Do you mean if the Keda service account doesn't have permissions? It will return an empty list.

Awesome, thanks!